blob/fileblob/fileblob.go

```diff
@@ -516,6 +516,47 @@ func (w *writer) Close() error {
 	return nil
 }
 
+// Copy implements driver.Copy.
+func (b *bucket) Copy(ctx context.Context, srcKey, dstKey string, opts *driver.CopyOptions) error {
+	// Note: we could use NewRangedReader here, but since we need to copy all of
+	// the metadata (from xa), it's more efficient to do it directly.
+	srcPath, _, xa, err := b.forKey(srcKey)
```

Comment:
is `xa` a conventional abbreviation/var name for metadata?

Reply:
I believe it's a shortening of the type (`type xattrs struct`), which I think is short for "extended attributes". `xa` is used in a number of other places in this file though. I'm open to better names, but probably in a different PR.